server.js

```diff
@@ -4098,6 +4141,10 @@ function deleteObject(objectKey) {
         utilities.deleteObject(objects[objectKey].name, objects, objectLookup, activeHeartbeats, knownObjects, sceneGraph, setAnchors);
     }
     // try to clean up any other state that might be remaining
+
+    if (objectKey.includes('_HUMAN_')) {
+        humanPoseFuser.removeHumanObject(objectKey);
+    }
```

Comment:
You might also need to add this logic to the `deleteObject` function in utilities.js (somehow we ended up with two deleteObject functions, which ideally would be consolidated into a single function)